### PR TITLE
PMM-10457 More friendly documentation for fresh contributors

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-10457

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Comment by gen1us2k** [comment]
I moved DBaaS related documentation to the [dbaas-controller repo](https://github.com/percona-platform/dbaas-controller/pull/341)

**Review by BupycHuk — APPROVED** [review]
Great job

**Review by palash25 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Welcome to Percona Monitoring and Management (PMM)!
 
-We're glad that you would like to become a Percona community member and participate in keeping open source open. [Percona Monitoring and Management (PMM)](https://www.percona.com/software/database-tools/percona-monitoring-and-management) is a open source database monitoring solution. It allows you to monitor your databases, different services (HAProxy, ProxySQL and etc) as well as Nodes, Kubernetes clusters and containers.
+We're glad that you would like to become a Percona community member and participate in keeping open source open. [Percona Monitoring and Management (PMM)](https://www.percona.com/software/database-tools/percona-monitoring-and-management) is a open source database monitoring solution. It allows you to monitor your databases, different services (HAProxy, ProxySQL and etc) as well as Nodes, Kubernetes clusters and containers. Please check our [Documentation](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html) for the actual architecture.
```
Comment (atymchuk) [inline CONTRIBUTING.md:3]: > We're glad that you would like to become a Percona community member
This seems to be too pretentious, we need to rephrase it.

Comment (atymchuk) [inline CONTRIBUTING.md:3]:
```suggestion
We'd be glad to welcome to Percona community which tries to keep the open source open. [Percona Monitoring and Management (PMM)](https://www.percona.com/software/database-tools/percona-monitoring-and-management) is an open source database monitoring solution. It allows you to monitor your databases, different services (HAProxy, ProxySQL and etc) as well as Nodes, Kubernetes clusters and containers. Please check our [Documentation](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html) for the actual architecture.
```

Comment (atymchuk) [reply CONTRIBUTING.md:3]: See en example below.

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (atymchuk) [reply CONTRIBUTING.md:3]: See an example below.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Welcome to Percona Monitoring and Management (PMM)!
 
-We're glad that you would like to become a Percona community member and participate in keeping open source open. [Percona Monitoring and Management (PMM)](https://www.percona.com/software/database-tools/percona-monitoring-and-management) is a open source database monitoring solution. It allows you to monitor your databases, different services (HAProxy, ProxySQL and etc) as well as Nodes, Kubernetes clusters and containers.
+We'd be glad to welcome to Percona community which tries to keep the open source open. [Percona Monitoring and Management (PMM)](https://www.percona.com/software/database-tools/percona-monitoring-and-management) is an open source database monitoring solution. It allows you to monitor your databases, different services (HAProxy, ProxySQL and etc) as well as Nodes, Kubernetes clusters and containers. Please check our [Documentation](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html) for the actual architecture.
```
Comment (atymchuk) [inline CONTRIBUTING.md:3]:
```suggestion
We'd be glad to welcome you to Percona community which tries to keep the open source open. [Percona Monitoring and Management (PMM)](https://www.percona.com/software/database-tools/percona-monitoring-and-management) is an open source database monitoring solution. It allows you to monitor your databases, different services (HAProxy, ProxySQL and etc) as well as Nodes, Kubernetes clusters and containers. Please check our [Documentation](https://docs.percona.com/percona-monitoring-and-management/details/architecture.html) for the actual architecture.
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (atymchuk) [reply CONTRIBUTING.md:3]: Just a minor change :)

**Review by atymchuk — APPROVED** [review]

**Pull request merged**